### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by kinggoesgaming**
This basically layout the basic layout of the project.

### CODE REVIEW

**Comment by kinggoesgaming** [comment]
should be good to go

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+Guidelines
+---
+
+Thank you for contributing to Foundations API! Have a read through before you start contributing.
```
Comment (Flibio) [inline CONTRIBUTING.md]: This isn't Foundations.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    * If possible add screenshots and console logs(as [pastebin] or [gists]) where necessary.
+* Requesting new features:
+    * Tell as what you would like to see.
+    * Provide justification/ use cases of how this is useful.
+* If the need be, we will ask for more information, so we have it right.
+
+### Pull requests
+Like to contribute some code? You are welcome! Just make sure you are aware of some things.
+
+#### Code Style
```
Comment (Flibio) [inline CONTRIBUTING.md]: We will use Sponge Code Style, just link to this page: https://docs.spongepowered.org/master/en/contributing/implementation/codestyle.html

File: README.md [hunk 3]
```diff
@@ -1 +1,46 @@
-# MinigameCoreAPI
\ No newline at end of file
+Minigame Core API
```
Comment (Flibio) [inline README.md]: To keep consistency, remove the space in MinigameCore.

File: .gitattributes [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+gradlew text eol=lf
+*.sh text eol=lf
+pre-commit text eol=lf
+
+*.java text
+*.java diff=java
+
+# Binary files that should not be modified
+*.jar binary
+*.png binary
```
Comment (Flibio) [inline .gitattributes]: Missing a few newlines at file ends.

**Pull request merged**